packages/edit-site/src/components/dataviews/view-actions.js

```diff
@@ -97,29 +90,25 @@ function PageSizeMenu( { view, onChangeView } ) {
 				>
 					{ /* TODO: probably label per view type. */ }
 					{ __( 'Rows per page' ) }
-				</DropdownSubMenuTriggerV2>
+				</DropdownMenuItemV2Ariakit>
 			}
 		>
 			{ PAGE_SIZE_VALUES.map( ( size ) => {
 				return (
-					<DropdownMenuItemV2
+					<DropdownMenuRadioItemV2Ariakit
 						key={ size }
-						prefix={
-							view.perPage === size && <Icon icon={ check } />
-						}
-						onSelect={ ( event ) => {
-							// We need to handle this on DropDown component probably..
-							event.preventDefault();
```

Comment:
While there is a new `hideOnClick` prop on the new version of `DropdownMenuItem`, using the `DropdownMenuRadioItem` component automatically keeps the menu open when clicked